web/templates/pages/tasks.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+from django import forms
+from .models import Task
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+		<div class="col-lg-6 col-sm-5">
+			<p>This page will show all tasks and who they are assigned to.</p>
+		</div>
+	</header>
+
+	<div class="row">
+	{% for task in task_list %}
+		<div class="col-md-3 col-xs-6">
+			<h3>{{ task.title }}</h3>
+			<p><strong>Status:</strong> {{ task.status }}</p>
+			<p><strong>Assigned to:</strong> {{ task.assigned_id }}</p>
+			<p><strong>Part of project:</strong>
+			{{ task.project_id }}</p>
```

Comment:
It would make more sense to display the title of the project and add a link to the project page. 

Hint: `{{ task.project_id.project_name }}`